README.md

```diff
@@ -34,6 +34,11 @@ Examples:
 
 (`host/managed` is for Foreman 1.2+, change to just `host` for Foreman 1.1)
 
+Plugins need to add the plugin name before the model name. For example:
+
+    ~/foreman/config/hooks/katello/product/before_create/01_integrate_catalog.sh
+    ~/foreman/config/hooks/foreman_tasks/task/dynflow_task/after_create/90_record_action.sh
```

Comment:
I don't think that's particularly true.  Anyway, using `.base_class` to identify the view might be best, then you can derive Task from DynflowTask while keeping the callback registered on DynflowTask specifically.